internal/configs/parser_config_dir.go

```diff
@@ -226,7 +229,36 @@ func (p *Parser) dirFiles(dir string, testsDir string) (primary, override, tests
 		}
 	}
 
-	return
+	return filterTfPathsWithTofuAlternatives(primary), filterTfPathsWithTofuAlternatives(override), filterTfPathsWithTofuAlternatives(tests), diags
+}
+
+func filterTfPathsWithTofuAlternatives(paths []string) []string {
```

Comment:
nit: rename it to something shorter? applyTofuExtOverride?

Reply:
I refrained from using the word override so people wouldn't confuse it with the Override File feature.